doc/api/packages.md

```diff
@@ -607,22 +608,28 @@ Runtimes or tools other than Node.js can use them at their discretion.
 
 These user conditions can be enabled in Node.js via the [`--conditions` flag][].
 
-The following condition definitions are currently endorsed by Node.js:
+The following condition definitions are currently endorsed by Node.js, and
+are _listed in order from most to least specific, as conditions should be
+used._
 
-* `"browser"` - any environment which implements a standard subset of global
-  browser APIs available from JavaScript in web browsers, including the DOM
-  APIs.
+* `"types"` - can be used by typing systems to resolve the typing file for
+  the given export, possible since the interface should be the same for all
+  variations. _This condition should always be included first._
+* `"deno"` - indicates a variation for the Deno platform.
```

Comment:
Is there something in the Deno docs we could link to? Like any mention of Deno’s support for `"exports"`, if it has any, or its Node.js compatibility mode?

Reply:
This is currently undocumented in Deno. Hence why I think it is important to highlight from a package compat perspective as it as much up to us to show how to make universal packages as anyone else.

Reply:
Is Deno unwilling to document it? It seems very strange that "not deno" would be deciding interop norms for deno.

Reply:
@ljharb Deno can and should document it, the point is that the dominant ecosystem can define the interop norms much better than the less popular platform alternatives.

There's been a lot of discussion I've yet to hear a strong argument for why we should _not_ include Deno to help users understand how conditions can be used to define universal packages. How does that materially cause any issues here?

Reply:
I think this is good to add, and there's nothing controversial about saying that the `deno` condition intends the Deno platform. I would expect them to do the same for us.

Reply:
I think it's quite controversial for node to declare the interop story for deno. If deno's decided it, then they should document it first, and we should link to it - but if they haven't documented it yet, then "not deno" should not be the first party to do so.

Reply:
I agree with @ljharb.

Reply:
From my understanding I also agree with @ljharb's last comment. Not that we should not document at some point but the first place being in the node repo does not sound right.

Reply:
We could submit a PR to Deno’s docs to suggest adding a mention of this. Then we would learn how they felt about it. If they merge in the PR, then we could add this reference here and link to their docs.

Reply:
Hi folks! I did an update to the manual that explains this bit in more detail and links back to Node's docs: https://deno.land/manual@main/npm_nodejs/compatibility_mode Let me know what you think, thanks!